test/drlg_l1_test.cpp

```diff
@@ -148,6 +148,20 @@ TEST(Drlg_l1, CreateL5Dungeon_hellfire_2_1180526547)
 	EXPECT_EQ(ViewPosition, Point(81, 45));
 }
 
+TEST(Drlg_l1, CreateL5Dungeon_hellfire_3_1369955278)
+{
+	LoadExpectedLevelData("hellfire/3-1369955278.dun");
+

```

Comment:
Players.resize(1);
MyPlayer = &Players[0];

Reply:
Arh implemented the test in a version before the breakage and this change and didn't think to look at the other test after rebasing :) Just got stomped when my local tests where passing...